spec/models/email_authentication_spec.rb

```diff
@@ -1,8 +1,46 @@
 require 'rails_helper'
 
 RSpec.describe EmailAuthentication, type: :model do
+  WEAK_PASSWORD_ERROR = "Please choose a stronger password with at least 8 characters. Try a mix of letters, numbers, and symbols."
+
   describe 'Associations' do
     it { should have_one(:user_authentication) }
     it { should have_one(:user).through(:user_authentication) }
   end
+
+  it "should require a non blank password" do
```

Comment:
Consider dropping the "should" - `it "requires...", it "allows..."`

Reply:
Updated. 👍